internal/delivery/http/route_boards_mock_test.go

```diff
@@ -7,11 +7,15 @@ import (
 )
 
 func (usecase *MockUsecase) GetBoardByID(ctx context.Context, boardID string) (bbs.BoardRecord, error) {
-	panic("Not implemented")
+	BoardRecord := NewMockBoardRecord("SYSOP", boardID, "嘰哩 ◎站長好!", true)
```

Comment:
```suggestion
	boardRecord := NewMockBoardRecord("SYSOP", boardID, "嘰哩 ◎站長好!", true)
```

在 golang 的區域變數，慣例上會以小寫開頭做變數命名。

另外在行數較短的時候，推薦使用比較短的命名方式例如 `record` 或是 `rec`, `r` 這樣

https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a